config/projects-to-test/openjdk19-excluded.files

```diff
@@ -29,32 +29,6 @@
   <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]unicode[\\/]FirstChar2.java$"/>
 </module>
 
-<!-- until https://github.com/checkstyle/checkstyle/issues/12507 -->
```

Comment:
>can add jdk19 to no-execution to make sure we can parse all record patterns that openjdk have in its test inputs ?

removal of related inputs from suppression is here

Reply:
@romani ping ^

Reply:
good.